packages/import-handler/src/index.ts

```diff
@@ -79,21 +109,30 @@ export const importHandler: EventFunction = async (event, context) => {
       return
     }
 
+    const regex = new RegExp('imports/(.*?)/')
+    const groups = regex.exec(data.name)
+    if (!groups || groups.length < 2) {
+      console.log('could not match file pattern: ', data.name)
+      return
+    }
+    const userId = [...groups][1]
+    if (!userId) {
+      console.log('could not extract userId from file name')
+      return
+    }
+
+    let countImported = 0
     await handler(stream, async (url): Promise<void> => {
       try {
         // Imports are stored in the format imports/<user id>/<type>-<uuid>.csv
-        const regex = new RegExp('imports/(.*?)/')
-        const groups = regex.exec(data.name)
-        if (!groups || groups.length < 2) {
-          console.log('could not match file pattern: ', data.name)
-          return
-        }
-        const userId = [...groups][1]
         const result = await importURL(userId, url, 'csv-importer')
         console.log('import url result', result)
+        countImported = countImported + 1
       } catch (err) {
         console.log('error importing url', err)
       }
     })
+
+    await importCompletedTask(userId, countImported)
```

Comment:
Should we send a different email if there is no import?

Reply:
Something like "Sorry but your import failed"

Reply:
Yeah, unfortunately the main issue is pulling the userId out of the file, so some cases we can't handle well :-/